Kitodo/src/main/java/org/kitodo/production/migration/NewspaperProcessesMigrator.java

```diff
@@ -619,6 +619,22 @@ public void createNextYearProcess() throws ProcessGenerationException, IOExcepti
         }
     }
 
+    private void createWorkpiece(Entry<String, IncludedStructuralElement> yearToCreate, String yearTitle, Process yearProcess) throws IOException {
```

Comment:
Different workpieces are assembled in this class. Could you rename this method to `createYearWorkpiece`?